config-sample.json

```diff
@@ -2,6 +2,8 @@
   "bridge": {
     "name": "Homebridge",
     "username": "CC:22:3D:E3:CE:30",
+    "manufacturer": "Homebridge.io",
```

Comment:
Could this be `homebridge.io` all lowercase?

Reply:
Done